recipes/civetweb/all/conanfile.py

```diff
@@ -118,8 +124,8 @@ def _configure_cmake(self):
         self._cmake.definitions["CIVETWEB_ENABLE_WEBSOCKETS"] = self.options.with_websockets
         self._cmake.definitions["CIVETWEB_SERVE_NO_FILES"] = not self.options.with_static_files
 
-        if tools.Version(self.version) >= "1.15":
-            self._cmake.definitions["CIVETWEB_ENABLE_ZLIB"] = self.options.with_zlib
+        if self.options.get_safe("with_zlib"):
+            self._cmake.definitions["CIVETWEB_ENABLE_ZLIB"] = True
```

Comment:
```suggestion
        if self._has_zlib_option:
            self._cmake.definitions["CIVETWEB_ENABLE_ZLIB"] = self.options.with_zlib
```

To be sure to have full control over `CIVETWEB_ENABLE_ZLIB`.